app/controllers/users/passwords_controller.rb

```diff
@@ -84,6 +86,16 @@ def handle_unsuccessful_password_reset
       render :edit
     end
 
+    def mark_profile_inactive
+      active_profile = resource.active_profile
+      return unless active_profile.present?
+      active_profile.update!(active: false)
+      analytics.track_event(
```

Comment:
they only happen together if the user has an active profile (LOA3).

if the user is LOA1 (no active profile) they will only get the password reset event.